### PR TITLE
Update to use Socket Mode as default 

### PR DESCRIPTION
**Pull request by srajiang**
###  Summary

Related to:
* Sibling changes in sample Bolt Python project: https://github.com/slackapi/bolt-python-getting-started-app/pull/1
* Related changes in Bolt for JS Getting Started documentation: https://github.com/slackapi/bolt-js/pull/990

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/bolt-js-getting-started-app/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

**Review by mwbrooks — CHANGES_REQUESTED** [review]
Looks good @srajiang!

I noticed that you have two copies of the same app - `app.js` (Socket Mode) and `app_http.js` (Request URL).

My concern is that our guides will drift apart and one (likely the `app_http.js`) will become out-of-date.

Do you think it would be better to only have `app.js`? The _Deploying to AWS Lambda Guide_ could add a step to disable Socket Mode. This is also a good education opportunity, since we may have developers who try to use Socket Mode on a Lambda-like deployment (which won't work).

**Comment by mwbrooks** [comment]
We'll also want to bump the version (major?) when we merge this PR https://github.com/slackapi/bolt-js-getting-started-app/tags

**Comment by srajiang** [comment]
@mwbrooks - I've updated the PR to include just the single `app.js` file w/ SocketMode as default + bumped the version.

File: .github/maintainers_guide.md [hunk 1]
```diff
@@ -13,8 +13,9 @@ All you need to work with this project is a supported version of [Node.js](https
 
 ### Updating
 
-This package is a dependency for our guides and other example apps.
-When making changes, it's important to keep all of our other guides and apps up-to-date.
+This package is a dependency for our guides and other example apps. There are many ways to configure a sample app. We prefer to keep a single version of `app.js` which should be consistent with the instructions and code blocks on the [Bolt JS Getting Started guide](https://slack.dev/bolt-js/tutorial/getting-started). This reduces the possibility of drift ocurring between multiple versions. 
+
+When making changes to this repo, it's also important to keep all of our other guides and apps up-to-date.
```
Comment (mwbrooks) [inline .github/maintainers_guide.md:18]: Nice! 👌🏻

**Review by mwbrooks — APPROVED** [review]
🎉 Yay! Thanks for giving the guides/template some ❤️ and updating them to Socket Mode!

**Comment by mwbrooks** [comment]
After merging, we'll want to throw a tag on the commit as well!

**Pull request merged**